docs/pages/components/panel.md

```diff
@@ -0,0 +1,161 @@
+{: .docs-intro}
+
+
+Use the panel if:
+<ul> 
+    <li>You need to group or display information.</li>
+    <li>You want to give users the option of hiding this information.</li>
+    <li>You want to show additional information on demand (for example, a panel could show optional input fields for an advanced search).</li>
+</ul>
+Do not use the panel if:
+<ul>
+    <li>You are designing an object page. Never use panels in the object page content area.</li>
+</ul>
+
+
+## Expandable Panel
+Expandable panels are much like fixed panels, except their content can be expanded and collapsed (including the info toolbar, if available).
+{% capture expand-panel %}
+<div class="fd-expand-panel">
+    <div class="fd-expand-panel__header">
+        <div class="fd-expand-panel__expand">
+            <button class="fd-button fd-button--transparent fd-expand-panel__button" aria-expanded="false" aria-haspopup="true" aria-controls="jhqDKYdf"></button>
+        </div>
+        <h5 class="fd-expand-panel__title">Panel Header</h5>
+        <div class="fd-toolbar fd-toolbar--clear fd-toolbar--transparent">
+            <span class="fd-toolbar__spacer fd-toolbar__spacer--auto"> </span>
+            <div class="fd-segmented-button" role="group" aria-label="Group label">
+              <button class="fd-button is-selected" aria-pressed="true">Left</button>
+              <button class="fd-button">Middle</button>
+              <button class="fd-button">Right</button>
+            </div>
+        </div>
+    </div>
+    <div role="region" aria-labelledby="jhqDKYdf" class="fd-expand-panel__content" aria-hidden="true" id="jhqDKYdf">
+        <span>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ut laoreet lorem. Vestibulum ante ipsum primis in faucibus orci luctus 
+            et ultrices posuere cubilia curae; Aenean sagittis aliquam justo et suscipit. 
+            Nam molestie, magna at elementum pulvinar, nisi enim venenatis ante, id convallis mi neque nec risus. Cras blandit sagittis augue at facilisis. 
+            Mauris egestas nunc nec diam mollis auctor. Vestibulum sed euismod elit, eget accumsan quam. Donec eleifend porttitor viverra. 
+            Nunc porttitor dictum erat at molestie. Sed quis velit dolor. Vestibulum et turpis eget enim gravida gravida vitae at massa. 
+            Suspendisse facilisis elit ut dolor posuere consectetur. Morbi ac nibh sit amet dolor lobortis tincidunt in ornare erat. 
+            Vestibulum tristique euismod enim, ac volutpat odio cursus sit amet.
+        </span>
+    </div>
+</div>
+{% endcapture %}
+{% include display-component.html component=expand-panel %}
+
+
+## Fixed Panel
+Fixed panels are useful for grouping custom content. They include headers and info toolbars.
+To create a fixed panel, add the `--fixed` modifier.
+
+{% capture fixed-panel %}
+<div class="fd-expand-panel fd-expand-panel--fixed">
```

Comment:
Not sure I am on board with this naming convention, because this panel doesn't expand at all